src/Integration.Vsix.UnitTests/ErrorList/SonarErrorListDataSourceTests.cs

```diff
@@ -327,49 +263,68 @@ public void RemoveFactory_ExistingSinkManagersAreNotified()
         }
 
         [TestMethod]
-        public void RemoveFactory_SelectedIssueIsInFactory_IssueDeselected()
+        public void Contains_NullArg_Throws()
```

Comment:
Should the `Contains` tests be in SonarErrorListDataSource_IssueLocationStoreTests.cs` instead?

Reply:
Good point, I'll move them